recipe/run_test.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+#
+echo -e "\n# Check installed directory structure"
+echo "# test -d ${PREFIX}/bin"
+test -d "${PREFIX}/bin"
+echo "# test -d ${PREFIX}/include"
+test -d "${PREFIX}/include"
+echo "# test -d ${PREFIX}/include/LHAPDF"
+test -d "${PREFIX}/include/LHAPDF"
+echo "# test -d ${PREFIX}/lib"
+test -d "${PREFIX}/lib"
+echo "# test -d ${PREFIX}/share"
+test -d "${PREFIX}/share"
+echo "# test -d ${PREFIX}/share/LHAPDF"
+test -d "${PREFIX}/share/LHAPDF"
+
+echo -e "\n# Check installed files"
+echo "# test -f ${PREFIX}/bin/lhapdf"
+test -f "${PREFIX}/bin/lhapdf"
+echo "# test -f ${PREFIX}/bin/lhapdf-config"
+test -f "${PREFIX}/bin/lhapdf-config"
+echo "# test -f ${PREFIX}/include/LHAPDF/LHAPDF.h"
+test -f "${PREFIX}/include/LHAPDF/LHAPDF.h"
+if [[ "${target_platform}" == linux-* ]]; then
+    echo "# test -f ${PREFIX}/lib/libLHAPDF.so"
+    test -f "${PREFIX}/lib/libLHAPDF.so"
+else
+    # macOS
+    echo "# test -f ${PREFIX}/lib/libLHAPDF.dylib"
+    test -f "${PREFIX}/lib/libLHAPDF.dylib"
+fi
+echo "# test -f ${PREFIX}/share/LHAPDF/lhapdf.conf"
+test -f "${PREFIX}/share/LHAPDF/lhapdf.conf"
+
+#
+echo -e "\n# Check lhapdf-config CLI API and flags return expected values"
+echo -e "# lhapdf-config --help\n"
+lhapdf-config --help
+
+echo -e "# lhapdf-config --version\n"
+lhapdf-config --version
+
+echo -e "\n# lhapdf-config --cxxflags: $(lhapdf-config --cxxflags)"
+
+echo -e "\n# lhapdf-config --ldflags: $(lhapdf-config --ldflags)"
+
+echo -e "\n# Check lhapdf CLI API"
+echo -e "# lhapdf --help\n"
+lhapdf --help
+
+#
+cd examples/
+echo -e "\n# Test example analyticpdf"
+"$CXX" analyticpdf.cc -o analyticpdf $(lhapdf-config --cxxflags --ldflags)
+./analyticpdf
```

Comment:
Currently failing on macOS with

```
# Test example analyticpdf
dyld[17941]: Library not loaded: '@rpath/libLHAPDF.dylib'
  Referenced from: '/Users/runner/miniforge3/conda-bld/lhapdf_1724108231610/test_tmp/examples/analyticpdf'
  Reason: tried: '/usr/local/lib/libLHAPDF.dylib' (no such file), '/usr/lib/libLHAPDF.dylib' (no such file)
```

so `rpath` is getting messed up somewhere.

Reply:
I don't actually understand how `libtool` works, but I think given some `git grep`ing in https://gitlab.com/hepcedar/lhapdf that this might(?) be related to `sys_lib_search_path_spec` in

https://gitlab.com/hepcedar/lhapdf/-/blob/c5b048f5794e483d85801d1ae39f42aaa46999dd/m4/libtool.m4#L2653-2666

Reply:
My first guess would be you need to use `$CXXFLAGS` as well.